src/hotspot/share/opto/superword.hpp

```diff
@@ -384,18 +384,6 @@ class PackSet : public StackObj {
   NOT_PRODUCT(static void print_pack(Node_List* pack);)
 };
 
-// ========================= SuperWord =====================
-
-// -----------------------------SWNodeInfo---------------------------------
-// Per node info needed by SuperWord
-class SWNodeInfo {
- public:
-  int         _alignment; // memory alignment for a node
```

Comment:
Note: `_alignment` is the last component left of the `SWNodeInfo`, we had already refactored away all other components and moved most of them to the `VLoopAnalylzer` submodules.